__tests__/app.test.js

```diff
@@ -118,6 +119,130 @@ describe("GET /api/articles", () => {
+    expect(articles).toBeSortedBy('votes', {ascending: true});
+  })
+ })
+ test("200: Responds with articles sorted by author in ascending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=author&order=asc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('author', {ascending: true});
+  })
+ })
+ test("200: Responds with articles sorted by author in descending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=author&order=desc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('author', {descending: true});
+  })
+ })
+ test("200: Responds with articles sorted by title in ascending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=title&order=asc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('title', {ascending: true});
+  })
+ })
+ test("200: Responds with articles sorted by title in descending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=title&order=desc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('title', {descending: true});
+  })
+ })
+ test("200: Responds with articles sorted by article_id in ascending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=article_id&order=asc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('article_id', {ascending: true});
+  })
+ })
+ test("200: Responds with articles sorted by article_id in descending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=article_id&order=desc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('article_id', {descending: true});
+  })
+ })
+ test("200: Responds with articles sorted by comment_count in ascending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=comment_count&order=asc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('comment_count', {ascending: true});
+  })
+ })
+ test("200: Responds with articles sorted by comment_count in descending order when specified", () => {
+  return request(app)
+  .get('/api/articles?sort_by=comment_count&order=desc')
+  .expect(200).then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    expect(articles).toBeSortedBy('comment_count', {descending: true});
+  })
+})
```

Comment:
These are all really good tests, but you dont need to have so many task cases for each valid column. Instead you could have a test for just a valid sort_by query, another test for a valid order query and then finally one for these two combined. Then when brought together by your error handling we have the confidence that this endpoint works as expected